src/SFA.DAS.EmployerIncentives.Api/Authentication/AuthenticationExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.EmployerIncentives.Configuration;
+
+namespace SFA.DAS.EmployerIncentives.Api.Authentication
+{
+    public static class AuthenticationExtensions
+    {
+        public static IServiceCollection AddApiAuthentication(this IServiceCollection services, IConfiguration config)
+        {
+            var azureActiveDirectoryConfiguration = config.GetSection(EmployerIncentivesConfigurationKeys.AzureActiveDirectoryApiConfiguration).Get<AzureActiveDirectoryApiConfiguration>();
```

Comment:
```suggestion
            var azureActiveDirectoryConfiguration = config.GetSection("AzureAd").Get<AzureActiveDirectoryApiConfiguration>();
```
You don't need to pass the whole path, to avoid the ConfigurationKeys constants you could change your config name and use `nameof(AzureActiveDirectoryApiConfiguration)'

I'm easy either way 😄